experimental/form_app/lib/src/http/mock_client.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:mockito/mockito.dart';
+
+class MockClient extends Mock implements http.Client {
```

Comment:
This is great. It's still simple enough for beginners, but way better than relying a real request to something external.